ArduPlane/servos.cpp

```diff
@@ -727,9 +727,35 @@ void Plane::servos_twin_engine_mix(void)
     }
 }
 
-
 /*
-  Set the flight control servos based on the current calculated values
+  Set throttle,attitude(in Attitude.cpp), and tilt servos for forced flare by RCx_OPTION switch for landing in FW mode
+  For Fixed Wind modes with manual throttle control only. Forces tilts up and throttle to THR_MIN.
+  Throttle stick must be in idle deadzone. This allows non-momentary switch to be used and quick bailouts
+  for go-arounds. Also helps prevent propstrike after landing with switch release on ground.
+*/
+void Plane::force_flare(void)
+{
+    if (!quadplane.in_transition() && !control_mode->is_vtol_mode() && !auto_throttle_mode && plane.channel_throttle->in_trim_dz() && plane.flare_switch_active) {
+        int32_t tilt = -SERVO_MAX;
```

Comment:
a comment explaining why the 3 cases for tilt would be good

Reply:
done